### PR TITLE
Add mentor bio for Alexhans

### PR DESCRIPTION
**Pull request by Alexhans**
I'm currently mentoring Python and C++ so I the same data to both language sections:

https://exercism.io/profiles/Alexhans

### CODE REVIEW

File: mentors/cpp/a.json [hunk 1]
```diff
@@ -14,5 +14,13 @@
         "link_url": null,
         "avatar_url": null,
         "bio": "Until now I have worked mainly with C. However, through my master in Embedded Systems I discovered the power of C++. I love the power and the flexibility of this language."
+    },
+    {
+    "github_username": "Alexhans",
```
Comment (SleeplessByte) [inline mentors/cpp/a.json]: Please fix the indentation. It should be 2 spaces more than the opening bracket `{`

File: mentors/python/a.json [hunk 2]
```diff
@@ -62,5 +62,13 @@
     "link_url": null,
     "avatar_url": null,
     "bio": "I use python a lot at work, and I really enjoy helping people to learn new skills!"
-  }
+  },
+  {
+    "github_username": "Alexhans",
```
Comment (SleeplessByte) [inline mentors/python/a.json]: Please fix the indentation. It should be 2 spaces more than the opening bracket `{`

**Comment by SleeplessByte** [comment]
Ha @rpottsoh 😆 this concurrency...

**Comment by rpottsoh** [comment]
Creepy.  Lol

**Comment by rpottsoh** [comment]
Is Travis CI stuck?

**Comment by Alexhans** [comment]
Hi @SleeplessByte and @rpottsoh.  If I understood the indentation rules correctly (2 spaces).  I have fixed the issues.  

There were other indentation problems besides the ones caused by my addition so I decided to fix them for consistency.

If you guys so desire.  I can create another pull request fixing the indentation for all the mentors using a script so there's no confusion from now on.

**Comment by rpottsoh** [comment]
@Alexhans there appears to be a conflict that needs to be resolved.

**Comment by Alexhans** [comment]
It seems I botched a `{`.  I'll fix it.  I now know to use rake before my commits.

**Comment by Alexhans** [comment]
I decided to just go back and re-edit from upstream/master to preserve a cleaner tree.

**Comment by SleeplessByte** [comment]
Welcome to Python and C++ 👋

**Pull request merged**